src/actions/feeds/show.cr

```diff
@@ -52,7 +52,7 @@ class Feeds::Show < FeedAction
           "link",
           rel: "related",
           type: "text/html",
-          href: Groups::Show.with(group).url
+          href: Groups::Bits::Index.with(group).url
```

Comment:
A little shortcut you can do is call `url` instead of `with`: `Groups::Bits::Index.url(group)`